IdentifiabilityODE.mpl

```diff
@@ -251,18 +273,17 @@ IdentifiabilityODE := proc(system_ODEs, params_to_assess, {p := 0.99, count_solu
   if infolevel > 1 then
     printf("%s %a\n", `Bound D_2 for assessing global identifiability: `, D2):
   end if:
-
   # (b, c) ---------
-  sample := SamplePoint(D2, x_vars, y_vars, u_vars, mu, X_eq, Y_eq, Q):
+  sample := SamplePoint(D2, x_vars, [op(y_vars), op(y_faux)], u_vars, mu, X_eq, Y_eq, Q):

```

Comment:
Maybe add `y_faux` to `y_vars` at the same place where we add the equations to `Y_eq` for consistency?